dev-tools/reconfigurator-cli/src/main.rs

```diff
@@ -890,7 +890,7 @@ fn cmd_blueprint_edit(
             let mut parent_sled_id = None;
             for sled_id in builder.sled_ids_with_zones() {
                 if builder
-                    .current_sled_zones(sled_id, BlueprintZoneFilter::All)
+                    .current_sled_zones(sled_id, |_disposition| true)
```

Comment:
Somewhat ugly, but I'll still take it as an alternative to retaining the `BlueprintZoneFilter`.